test/testdata/maintidx_under_100.go

```diff
@@ -0,0 +1,197 @@
+// args: -Emaintidx
+// config_path: testdata/configs/maintidx_under_100.yml
+package testdata
+
+func over20() { // ERROR "Function name: over20, Cyclomatic Complexity: 1, Halstead Volume: 8.00, Maintainability Index: 86"
```

Comment:
Is error expected here? (with setting `under:20` )?

Reply:
This error is expected because we have set `under:100` in `testdata/configs/maintidx_under_100.yml`.

https://github.com/golangci/golangci-lint/pull/2435/files#diff-231948d315b367969bc59f5d0e7c86deb00e6441f8e860dc790ac7dbb250ba51